src/core/stdcpp/string_view.d

```diff
@@ -123,6 +123,14 @@ private:
         const value_type* __data;
         size_type __size;
     }
+    else version (CppRuntime_Sun)
+    {
+        size_t _M_len;
+        const(T)* _M_str;
+
+        alias __data = _M_str;
+        alias __size = _M_len;
```

Comment:
Developer Studio doesn't implement this, so maybe add a pragma saying as much. Falling back on doing what gcc does is the safe option, which seems to be what you're doing here.